README.rst

```diff
@@ -20,11 +20,18 @@ Use it like this:
         # language_info.name in a kernel_info_reply should match this
         language_name = "dg"
 
+        # language_info.file_extension, should match kernel_info_reply
+        # (and start with a dot)
+        file_extension = ".py"
```

Comment:
The example in the README was originally for [dogelang](https://pyos.github.io/dg/), to highlight that it's not just for Python. I think it's now become a bit of a confusing hodgepodge of that and Python, and dogelang itself has probably changed since I wrote the example (since it's not exactly a very serious thing).

That can be cleaned up in a separate PR, though.